### PR TITLE
Refix #210

### PR DESCRIPTION
**Pull request by exKAZUu**
#210
I've conducted the following tests and confirmed that there is no inconsistency.

### CODE REVIEW

**Comment by dr-dimitru** [comment]
Are you sure it's good idea to remove checks?

**Comment by exKAZUu** [comment]
Actually, I think we should keep checks even if some inconsistent behaviour exists because checks' errors are useful for debugging.
In my use case, I'd like to expect `find/findOne(), find/findOne({}), find/findOne(null), find/findOne(undefined)` works as well as `MongoCollection`.
How do you think adding the following code in `find` and `findOne`:
```
check selector, Match.Optional Match.OneOf Object, String, null, undefined
check options, Match.Optional Object
```

Meteor API document says that the type of `selector` is `Mongo Selector`, `Object ID`, or `String`. So I think we should accept only `Object`, `String`, `null` and `undefined`.

**Comment by exKAZUu** [comment]
I've added check statements in this PR.

File: files.coffee [hunk 1]
```diff
@@ -1458,7 +1458,7 @@ class FilesCollection
   ###
   find: (selector, options) ->
     console.info "[FilesCollection] [find(#{JSON.stringify(selector)}, #{JSON.stringify(options)})]" if @debug
-    check selector, Match.OneOf Object, String, null
+    check selector, Match.Optional Match.OneOf Object, String, null
```
Comment (dr-dimitru) [inline files.coffee]: @exKAZUu have you run all test after this changes?
Looks like it will fail on *Boolean* values

Comment (exKAZUu) [inline files.coffee]: Meteor API document says that the type of selector is Mongo Selector, Object ID, or String.
So I think we should accept only Object, String, null and undefined.

Should we accept `Boolean` values?

**Comment by exKAZUu** [comment]
Thank you for your comment.
I updated the comparison table https://github.com/VeliovGroup/Meteor-Files/pull/224#issuecomment-245797874

**Comment by exKAZUu** [comment]
@dr-dimitru If you want me to fix something, could you tell me it?

Please note that we cannot satisfy with both `check` and the complete consistency with Meteor's `find` and `findOne`.
Personally, I'd like to keep `check` and break little consistency, but I'd like to pass `null` or `undefined` to `FilesCollection.find` and `FilesCollection.findOne` as well as Meteor's ones in order to avoid `null` and `undefined` checks.

**Comment by dr-dimitru** [comment]
@exKAZUu everything looks good to me, working on the new release now.
Your PR will be merged

**Comment by exKAZUu** [comment]
I see. Thanks!

**Pull request merged**